Sources/Helpers/OAAppSettings.h

```diff
@@ -1029,6 +1029,7 @@ typedef NS_ENUM(NSInteger, EOARateUsState)
 @property (nonatomic) NSString *customPluginsJson;
 
 @property (nonatomic) OATravelGuidesState *travelGuidesState;
+@property (nonatomic) NSArray<UIViewController *> *previousOpenedScreens;
```

Comment:
Not the best approach and place for storing an array of controllers

Reply:
I know it :( 

But what should I use instead it? How can I open several view controllers at once?

By example: I want to open 10 vc. But without 10 opening animations. (It will be so long). Animate appearing only for last vc. And all previous vc just add to history of navigation controller. Is there any other possibilities to do that?